lib/systems/examples.nix

```
@@ -221,6 +221,28 @@ rec {
     libc = "newlib";
   };
 
+  #
+  # UEFI
+  #
+
+  i686-uefi = {
+    config = "i686-unknown-windows-gnu";
+    rust.config = "x86_64-unknown-uefi";
```

Should this be [`i686-unknown-uefi`](https://github.com/rust-lang/rust/blob/master/compiler/rustc_target/src/spec/i686_unknown_uefi.rs)?

```suggestion
    rust.config = "i686-unknown-uefi";
```

---

definitely right